examples/gfpgan/model/restoration_model.py

```diff
@@ -0,0 +1,120 @@
+import logging
+import tempfile
+
+import cv2
+import numpy as np
+import requests
+from basicsr.archs.rrdbnet_arch import RRDBNet
+from gfpgan import GFPGANer
+from model.utils import upload_file_to_s3
+from PIL import Image
+from realesrgan import RealESRGANer
+
+logger = logging.getLogger(__name__)
+
+bg_tile = 400  # Tile size for background sampler, 0 for no tile during testing
```

Comment:
Very minor: Should we upper case these constants. I'm just worried that it won't match the style in rest of the repo.